chemprop/features/features_generators.py

```diff
@@ -132,8 +132,16 @@ def rdkit_2d_normalized_features_generator(mol: Molecule) -> np.ndarray:
         raise ImportError('Failed to import descriptastorus. Please install descriptastorus '
                           '(https://github.com/bp-kelley/descriptastorus) to use RDKit 2D normalized features.')
 except AttributeError as e:
-    raise e('`descriptastorus==2.6.1` is incompatible with `scipy<1.9`. Please try changing '
-                         'descriptastorus versions or update scipy to avoid issues with `scipy.stats.gibrat`.')
+    from packaging import version
```

Comment:
what is `packaging`? I don't think we need to add a 3rd party package just for this obscure error message. I would just print the version information with a note saying that certain version combos don't work (like you already have).

Reply:
`packaging` is used by `setuptools` [ref](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python). I agree that just printing the version information is simpler than checking it.

Reply:
we don't rely on `setuptools` though--only our build system does (which is different from the actual environment in which `chemprop` runs). It's a very minor issue, but it would entail adding _another_ dependency for just 1 line of code.